README.md

```diff
@@ -52,7 +52,7 @@ libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/ma
 
 * [ASP.NET Core](./src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
 * [Grpc.Net.Client](./src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md)
-* [HTTP clients](./src/OpenTelemetry.Instrumentation.Http/README.md)
+* [HTTP client](./src/OpenTelemetry.Instrumentation.Http/README.md)
```

Comment:
I don't have strong opinion on this, the actual instrumentation lib seems to support two clients?

Reply:
In that case, we could be more specific:

```suggestion
* [HttpClient and HttpWebRequest](./src/OpenTelemetry.Instrumentation.Http/README.md)
```

Reply:
Does that look good?

Reply:
I'm slightlying leaning towards full name (System.Net.Http.HttpClient, System.Net.HttpWebRequest) - it's the same as what we did for gRPC client. No strong opinion though.

Reply:
Probably change the next line (SQL client) as well.